sce.sh

```diff
@@ -225,12 +187,12 @@ then
     then
         print_missing_config $REPO_LOCATION $missingPaths
     fi
-    if [ $start_only_mongodb_container -eq 0 ]
+    if [ $start_only_mongodb_container == 0 ]
     then
         docker-compose -f docker-compose.dev.yml up mongodb -d
         exit 0
     fi
-    if [ $name == $SCE_DISCORD_BOT_REPO_NAME ]
+    if [ $name == "SCE-discord-bot" ]
```

Comment:
undo?

Reply:
I renamed all the repo variables into strings so this should be fine.